fdbrpc/IRateControl.h

```diff
@@ -37,9 +37,8 @@ class IRateControl {
 // An IRateControl implemenation that allows at most hands out at most windowLimit units of 'credit' in windowSeconds seconds
 class SpeedLimit : public IRateControl, ReferenceCounted<SpeedLimit> {
 public:
-	SpeedLimit(int windowLimit, int windowSeconds) : m_limit(windowLimit), m_seconds(windowSeconds), m_last_update(0), m_budget(0) {
-		m_budget_max = m_limit * m_seconds;
-		m_last_update = timer();
+	SpeedLimit(int windowLimit, double windowSeconds) : m_limit(windowLimit), m_seconds(windowSeconds), m_last_update(0), m_budget(0) {
+		m_last_update = now();
```

Comment:
why changing timer() to now()? 

Do you intentionally want to not increase time between "wait"s?

Reply:
I remember the `timer()` will add non-determinism in simulation tests.